### PR TITLE
Phase1 backend

### PR DESCRIPTION
**Pull request by MongoCaleb**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-23972

### Staging
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/phase1_backend/tutorial/

### CODE REVIEW

File: source/tutorial.txt [hunk 1]
```diff
@@ -55,33 +42,18 @@ Next, complete any of the following clients to work with your new backend:
...
-
-Realm template applications are working example apps you can explore and 
-modify as you learn about Realm. These apps are a good choice for developers
-who prefer to experiment directly with working code instead of reading a 
-guide that walks you through implementing the code.
+Before You Start
+----------------
 
-Check out the :ref:`Realm template apps <template-apps>`.
+These tutorials start with the Flexible Sync Template App for your chosen language
```
Comment (dacharyc) [inline source/tutorial.txt:47]: Will this be true for all tuts? I'm not sure we scoped the CLI tutorial as part of this phase 1 project. I could be wrong, though!

File: source/tutorial.txt [hunk 2]
```diff
@@ -6,46 +6,33 @@ Tutorials
...
-Realm. 
+The Realm tutorials provide step-by-step guidance on building Realm apps. Each 
+of the tutorials starts with the 
+:ref:`Flexible Sync Template App <template-apps>` and then adds 
+functionality to it.  
 
-For more details, check out 
 
-- :doc:`What you will build <tutorial/preview>`
+.. button:: Check out the Template Apps
```
Comment (dacharyc) [inline source/tutorial.txt]: So this button saying "Check out the Template Apps" right below a link for "Flexible Sync Template App" struck me as odd. I was slightly confused about which thing I should click. I like the idea of providing both the docs and a CTA button here but we should probably be more explicit about the link contents so users have some idea what they're getting when they click. Maybe the docs link could say something like "Learn more about the Flexible Sync Template App that this tutorial builds on" for the docs link, and then maybe "Create a Template App" or "Build a Template App" or something that's a slightly clearer CTA than "check out" for the button?

**Review by dacharyc — APPROVED** [review]
Overall, I think this is fine; just a couple of small comments. But I do think we lose something by not mentioning that it's a task tracker app. The first template app on the linked docs page is "Manage Database Views" which seems to be trigger-related, so as a user I think I can't safely infer what we'll be building. And I just realized that the descriptions for most of the todo apps don't actually say they're todo apps - just a very generic language/platform description like:

> Android app written in Kotlin using the [Realm Java SDK](https://www.mongodb.com/docs/realm/sdk/java/#std-label-java-intro) to sync data to App Services's server. Uses Flexible Sync.

We should probably clean up those descriptions. Maybe it's time to think about dropping the PBS templates from the list if Product really wants us to discontinue PBS as the list of templates is quite long now.

That stuff is probably outside the scope of this PR but we should consider it as part of the context for this content, and the larger tutorial project.

File: source/tutorial.txt (hunk 1 above)
Comment (MongoCaleb) [reply source/tutorial.txt:47]: Everything but Web supports flex sync now, and we have template apps for each of them. So I think this is legit to say. 
Created ticket to make changes to the other repo's page on the Template Apps.

**Pull request merged**